src/index.html

```diff
@@ -1,15 +1,194 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="" onsubmit="onSubmit()">
+      <fieldset class="fieldset-section">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="form-field">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </label>
```

Comment:
delete this empty line between text and input everywhere, it's unnecessary here

```suggestion
        <label for="surname" class="form-field">
          Surname:
          <input
            type="text"
            name="surname"
            id="surname"
          >
        </label>
```